grouper/fe/handlers/user_metadata.py

```diff
@@ -92,6 +98,7 @@ def post(self, *args: Any, **kwargs: Any) -> None:
                 form=form,
                 user=user,
                 metadata_key=metadata_key,
+                set_enabled=known_field,
```

Comment:
I find `set_enabled` to be a bit unintuitive as a name. Would `is_enabled` be a better fit?

Reply:
(Probably my issue is with how overloaded the word `set` is)

Reply:
Yeah I agree, I struggled a bit with naming that. I like `is_enabled` better.